docs/api/en/animation/AnimationMixer.html

```diff
@@ -102,7 +102,8 @@ <h3>[method:undefined uncacheRoot]([param:Object3D root])</h3>
 		<p>
 			Deallocates all memory resources for a root object. Before using this
 			method make sure to call [page:AnimationAction.stop]() for all related
-			actions.
```

Comment:
How about shorten this a bit:

> Before using this method make sure to call [page:AnimationAction.stop]() for all related actions or alternatively call [page:.stopAllAction]().


> which is convenient when the mixer operates on a single root.

An instance of animation mixer should always have a single root and that reference should not be changed during the usage. There is only `getRoot()` and no corresponding setter.

Reply:
This is a little confisuing because other methods accept options roots. If there is always one root, why do other methods allow passing in roots?

Reply:
You are right, I've indeed overlooked the `optionalRoot` parameters. In this case, the sentence is okay!

Reply:
Awesome